fivetran/resource_connector.go

```diff
@@ -301,6 +301,7 @@ func resourceConnectorSchemaConfig() *schema.Schema {
 				"client_name":           {Type: schema.TypeString, Optional: true},
 				"domain_type":           {Type: schema.TypeString, Optional: true},
 				"connection_method":     {Type: schema.TypeString, Optional: true},
+				"is_single_table_mode":  {Type: schema.TypeString, Optional: true},
```

Comment:
```suggestion
				"is_single_table_mode":  {Type: schema.TypeString, Optional: true, Computed: true},
```
It is boolean field in connector credentials and if not set directly it would be returned in response.
Provider will detect unwanted changes in resource state while reading after create/update.
To prevent such situations all non-nullable fields should be marked as computed (usually it is boolean or enum fields). 
And let's move it here: 
https://github.com/fivetran/terraform-provider-fivetran/blob/main/fivetran/resource_connector.go#L129